### PR TITLE
Update the documentation of the Timers module

### PR DESCRIPTION
**Pull request by LaszloLango**
IoT.js-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by LaszloLango** [comment]
Related issue: #873

File: docs/api/IoT.js-API-Timers.md [hunk 1]
```diff
@@ -9,61 +9,84 @@ The following shows timer module APIs available for each platform.
...
-## Prototype Functions
+Schedules execution of a one-time `callback` after `delay` milliseconds. Returns a `Timeout` for use with `clearTimeout()`. If `callback` is not a function, a `TypeError` will be thrown.
 
-### `activate()`
+Example:
 
-When called, requests that the IoT.js event loop not exit so long as the Timeout is active.
+```javascript
+var timeout = setTimeout(function() {
+  // Do something what will be executed after one second.
```
Comment (zherczeg) [inline docs/api/IoT.js-API-Timers.md]: which will be executed

File: docs/api/IoT.js-API-Timers.md [hunk 2]
```diff
@@ -9,61 +9,84 @@ The following shows timer module APIs available for each platform.
...
+}, 1000);
+```
+
+### clearTimeout(timeout)
+
+* `timeout` {Timeout} A Timeout object as returned by `setTimeout()`.
```
Comment (zherczeg) [inline docs/api/IoT.js-API-Timers.md]: Timeout object returned by

File: docs/api/IoT.js-API-Timers.md [hunk 3]
```diff
@@ -9,61 +9,84 @@ The following shows timer module APIs available for each platform.
...
-### `close()`
+### setInterval(callback, delay[, args..])
+* `callback` {Function} The function to call when the timer elapses.
+  * `...args` {any}
+* `delay` {number} The number of milliseconds to wait before calling the `callback`.
+* `...args` {any} - Optional arguments to pass when the `callback` is called.
+* Returns: {Timeout}
 
-When called, the active Timeout object will not require the IoT.js event loop to remain active. If there is no other activity keeping the event loop running, the process may exit before the Timeout object's callback is invoked.
+Schedules repeated execution of `callback` every `delay` milliseconds. Returns a `Timeout` for use with `clearInterval()`. If `callback` is not a function, a `TypeError` will be thrown.
```
Comment (zherczeg) [inline docs/api/IoT.js-API-Timers.md]: `Timeout` object

File: docs/api/IoT.js-API-Timers.md [hunk 4]
```diff
@@ -9,61 +9,84 @@ The following shows timer module APIs available for each platform.
...
+Example:
 
-### `setTimeout(callback, delay[, args..])`
-* `callback <Function([args..])>`
-* `delay <Number>`
-* `...args <any>` - Optional arguments to pass when the callback is called.
-* Returns: `<Timeout>`
+```javascript
+var timeout = setInterval(function() {
+  // Do something what will be executed repeatadly one time per second.
```
Comment (zherczeg) [inline docs/api/IoT.js-API-Timers.md]: which

File: docs/api/IoT.js-API-Timers.md [hunk 5]
```diff
@@ -9,61 +9,84 @@ The following shows timer module APIs available for each platform.
...
+```
+
+## Class: Timeout
+
+This object is created internally and is returned from `setTimeout()` and `setInterval()`.
+
+
+### timeout.activate()
+
+When called, requests that the IoT.js event loop not exit so long as the Timeout is active.
```
Comment (zherczeg) [inline docs/api/IoT.js-API-Timers.md]: event loop should not exit as long as

File: docs/api/IoT.js-API-Timers.md [hunk 6]
```diff
@@ -9,61 +9,84 @@ The following shows timer module APIs available for each platform.
...
 
-### `clearInterval(intervalObject)`
-* `intervalObject <Timeout>`
+### timeout.close()
 
-Disables a `Timeout` object from `setInterval` method.
+When called, the active `Timeout` object will not require the IoT.js event loop to remain active. If there is no other activity keeping the event loop running, the process may exit before the `Timeout` object's callback is invoked.
```
Comment (zherczeg) [inline docs/api/IoT.js-API-Timers.md]: will not force the IoT.js event loop
If there are no other scheduled activites, the process may exit...

File: docs/api/IoT.js-API-Timers.md [hunk 7]
```diff
@@ -9,61 +9,84 @@ The following shows timer module APIs available for each platform.
...
+...
+clearInterval(timeout);
+```
+
+## Class: Timeout
+
+This object is created internally and is returned from `setTimeout()` and `setInterval()`.
+
+
+### timeout.activate()
```
Comment (daeyeon) [inline docs/api/IoT.js-API-Timers.md]: I suggest renaming activate() to ref() and close() to unref() for consistency.

Comment (LaszloLango) [inline docs/api/IoT.js-API-Timers.md]: Sounds good. I'll do it.

Comment (LaszloLango) [inline docs/api/IoT.js-API-Timers.md]: done

**Pull request merged**